examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
+===========================
+In this example we are going to look at how pixel and map coodinates work with WCS Axes.
+"""
+
+import matplotlib.pyplot as plt
+
+from astropy import units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+###############################################################################
+# Lets start by creating a Map from the sample data.
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###############################################################################
+# Lets choose xlimits and ylimits in arcsecs that we are interested in.
+xlims_world = [500, 1100]*u.arcsec
+ylims_world = [-800, 0]*u.arcsec
+
+###############################################################################
+# By setting the projection of the axis, a WCSAxes is created and this enables
+# the plot to be created with the map coordinates, in this case helioprojective
+# coordinates. The standard plot
+# command expects pixel coordinates. It is also possible to plot a point using the
+# map coordinates if we pass the transformation but this transform expects
+# the values to be in degrees and not arcseconds.
```

Comment:
This paragraph is from this example https://docs.sunpy.org/en/stable/generated/gallery/plotting/wcsaxes_map_example.html#sphx-glr-generated-gallery-plotting-wcsaxes-map-example-py
and should probably be changed such that it is a different example?

Reply:
This may just mean being more explicit about how there is a difference between pixel and world coordinates for a user that isn't familiar with the terms

Reply:
Hey @hayesla yeah i took that comment as we were showing all three together ( pixel and map coord as well). But you are right i will look into it. Also any suggestions on the same will be great.